Name: Disable Various Phone Sounds
Version: 1.3.5-1
Description: disable end call, dropped call, missed call, ignored call sounds. The missed call will appear as a notifification. Updated to disable dropped call sound.

--- .orig/usr/palm/applications/com.palm.app.phone/app/controllers/announcer-assistant.js
+++ /usr/palm/applications/com.palm.app.phone/app/controllers/announcer-assistant.js
					 

@@ -322,7 +323,7 @@
 					"height": height, 
 				}, 
 				this.onAnnounceMissedCreated.bind(this, name, number, picture, contact, Messages.missedMessage, timeStamp),
-				'popupalert');
+				'dashboard');
 		}
 
 	},
@@ -442,7 +443,7 @@
 		this.appControl.showBanner(
 			{
 				"messageText": message,
-				"soundClass": 'notifications'
+				"soundClass": ''
 			}, 
 			
 			{
@@ -796,7 +797,7 @@
 					applicationStylesheets: ["stylesheets/phone.css"],
 					name: stageName, 
 					"height": height, 
-					soundclass: "alerts"
+					soundclass: "none"
 				}, 
 				this.onAnnounceDisconnectAbnormalCreated.bind(this, name, callState.number, label, picture, message, callerDetails),
 				'popupalert');
@@ -868,7 +869,7 @@
 		this.appControl.showBanner(
 			{
 				messageText: message,
-				soundClass: 'notifications'
+				soundClass: ''
 			}, 
 			{
 				prefill: callbackNumber
@@ -903,7 +904,7 @@
 					applicationStylesheets: ["stylesheets/phone.css"],
 					name: stageName, 
 					"height": height, 
-					soundclass: "alerts"
+					soundclass: "none"
 				}, 
 				this.onAnnounceDialFailCreated.bind(this, message),
 				'popupalert');
